tools/exporter/tests/test_csv_exporter.py

```diff
@@ -0,0 +1,22 @@
+# coding=utf-8
+# Author: SafetyCulture
+# Copyright: © SafetyCulture 2016
+
+import os
+import sys
+import unittest
+import json
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', 'exporter'))
+import csvExporter as csv
+
+class ExporterTestCase(unittest.TestCase):
+
+    def test_unit_one(self):
```

Comment:
def test_convert_single_question_with_standard_yes_no_na_answered_yes(...):
and remove the comment
also rename the expected output file, hard to understand what test_1 means.